app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -6,7 +6,9 @@ def create_in_provider(manager_id, params)
       params = provider_params(params) if respond_to?(:provider_params)
       manager = ExtManagementSystem.find(manager_id)
       tower_object = provider_collection(manager).create!(params)
-
+      if respond_to?(:refresh_in_provider)
```

Comment:
I like this better ... The full refresh can stay for now until we have @agrare's [`refresh_new_target`](https://github.com/ManageIQ/manageiq/pull/14247) in place.

And, this avoids doing the double refresh *just* to make sure we get the updated Tower git repo.